packages/backend/src/models/DashboardModel/DashboardModel.test.ts

```diff
@@ -145,111 +143,79 @@ describe('DashboardModel', () => {
     });
 
     test('should create dashboard with tile ids', async () => {
-        const spaceUuid = 'space uuid';
+        tracker.on.select(SpaceTableName).responseOnce([spaceEntry]);
```

Comment:
Updated failing unit tests. Query arguments/bindings are now checked in the expect rather than the mocks as it produces readable error messages.